src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -101,7 +103,8 @@ public void parse_invalidValue_failure() {
         assertParseFailure(parser, "1" + TAG_EMPTY + TAG_DESC_FRIEND + TAG_DESC_HUSBAND, Tag.MESSAGE_CONSTRAINTS);
 
         // multiple invalid values, but only the first invalid value is captured
-        assertParseFailure(parser, "1" + INVALID_NAME_DESC + INVALID_EMAIL_DESC + VALID_ADDRESS_AMY + VALID_PHONE_AMY,
+        assertParseFailure(parser, "1" + INVALID_NAME_DESC + INVALID_EMAIL_DESC
+                        + VALID_ADDRESS_AMY + VALID_PHONE_AMY,
```

Comment:
Similar to the above comment, VALID_ADDRESS_AMY should be VALID_MODULE_CODE_AMY.